### PR TITLE
Release 1.2.3

### PR DESCRIPTION
**Pull request by diasdm**
- [FP-2754](https://movai.atlassian.net/browse/FP-2754): Control + S doesn't save flow
- [FP-2653](https://movai.atlassian.net/browse/FP-2653): Nodes disappear after updating a flow parameter in tree view mode

[FP-2754]: https://movai.atlassian.net/browse/FP-2754?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FP-2653]: https://movai.atlassian.net/browse/FP-2653?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/editors/Flow/view/Core/Graph/GraphBase.js [hunk 1]
```diff
@@ -270,26 +270,38 @@ export default class GraphBase {
...
-      }
-    });
-    // Update links
-    this.updateLinks(data.Links || {});
-    this.nodeStatusUpdated();
-    // Update exposed ports
-    this.loadExposedPorts(data.ExposedPorts || {}, true);
-    // Let's re-validate the flow
-    this.validateFlow();
+    if (this.viewMode === FLOW_VIEW_MODE.default) {
```
Comment (quirinpa) [inline src/editors/Flow/view/Core/Graph/GraphBase.js:273]: If you did:
```javascript
if (this.viewMode !== FLOW_VIEW_MODE.default)
  return;
  
  // else code
```

There would be less of a diff to review

File: src/editors/Flow/view/Flow.jsx [hunk 2]
```diff
@@ -1380,13 +1384,6 @@ export const Flow = (props, ref) => {
         if (viewMode === FLOW_VIEW_MODE.treeView && docData.doc.name === name) {
           const subFlows = mainInterfaceRef.current.graph.subFlows;
 
-          if (!docData.thisDoc.isDirty) {
```
Comment (quirinpa) [inline src/editors/Flow/view/Flow.jsx:1383]: This is removed on purpose? Why?

**Comment by diasdm** [comment]
@quirinpa these changes have been reviewed in #322, any questions to code changed in it pls do them there

**Comment by quirinpa** [comment]
That's already merged. I can only pose questions and expect an answer on PR's that haven't.

File: src/editors/Flow/view/Flow.jsx (hunk 2 above)
Comment (duartecoelhomovai) [reply src/editors/Flow/view/Flow.jsx:1383]: The if was not making sense. If no changes identified then safe. Some dead code right there. If you can justify me why the code was there or its intention i would add it back.

Comment (quirinpa) [reply src/editors/Flow/view/Flow.jsx:1383]: Oh I see what you mean. Thanks

**Review by quirinpa — DISMISSED** [review]

**Review by manuelnogueiramov — APPROVED** [review]

**Review by quirinpa — APPROVED** [review]

**Pull request merged**